ddl/ddl_worker.go

```diff
@@ -537,10 +540,30 @@ func (w *worker) runDDLJob(d *ddlCtx, t *meta.Meta, job *model.Job) (ver int64,
 
 		job.Error = toTError(err)
 		job.ErrorCount++
+		// Load global ddl variables.
+		if err1 := loadDDLVars(w); err1 != nil {
```

Comment:
I think a config item is better.

Reply:
This Limit will not change in general.

Reply:
em... I think the variable in the config is some system machine variable such as log file, cache size, max_connection ... 

The disadvantage of config is you should restart the TiDB or add an http api to change it, and the http api change only takes effect in one machine.
But the variable of `tidb_ddl_error_count_limit` is not a `machine` variable, it only takes effect in TiDB owner. So I think global variable is better.

If the variable is not changed in general, so why we need this variable? Just hardcode to `const` will be better. 😃